### ISSUE TITLE
WindowsAppSDK v 1.5 Breaks WACK

### ISSUE THREAD

**Issue by greatoceansoftware** [microsoft/microsoft-ui-xaml#9478]
### Describe the bug

Publication of my WinUI 3 app to Microsoft Store MSIX works fine with v1.4 of the Windows App SDK. When updating to v1.5.240311000, it no longer passes the Windows Runtime metadata validation section of the WACK test. This is the only variable that has changed. Same code base. See the before/after screen shots below.

### Steps to reproduce the bug

Unfortunately, the project is large and proprietary, so I can't upload it. However, if it helps, these are the Usings. See the screenshots below for WACK test with WindowsAppSDK v 1.4 vs. Windows AppSDK 1.5.

```
global using Code;
global using CsvHelper;
global using CsvHelper.Configuration.Attributes;
global using Microsoft.UI;
global using Microsoft.UI.Input;
global using Microsoft.UI.Xaml;
global using Microsoft.UI.Xaml.Controls;
global using Microsoft.UI.Xaml.Controls.Primitives;
global using Microsoft.UI.Xaml.Data;
global using Microsoft.UI.Xaml.Input;
global using Microsoft.UI.Xaml.Media;
global using Microsoft.UI.Xaml.Media.Animation;
global using Microsoft.UI.Xaml.Navigation;
global using Nager.Date;
global using System;
global using System.Collections;
global using System.Collections.Generic;
global using System.Collections.ObjectModel;
global using System.ComponentModel;
global using System.Diagnostics;
global using System.Globalization;
global using System.IO;
global using System.Linq;
global using System.Net.Http;
global using System.Net.Http.Json;
global using System.Reflection;
global using System.Runtime.CompilerServices;
global using System.Security.Cryptography;
global using System.Text;
global using System.Text.Json;
global using System.Text.Json.Serialization;
global using System.Text.RegularExpressions;
global using System.Threading.Tasks;
global using System.Timers;
global using System.Windows.Input;
global using Windows.ApplicationModel.DataTransfer;
global using Windows.ApplicationModel.Resources;
global using Windows.Globalization.DateTimeFormatting;
global using Windows.Graphics;
global using Windows.Services.Store;
global using Windows.Storage;
global using Windows.Storage.Pickers;
global using Windows.System;
global using Windows.System.UserProfile;
global using Windows.UI;
global using Windows.UI.Core;
global using Windows.UI.Text;
global using WinRT.Interop;
```

### Expected behavior

Pass all WACK tests. Not concerned about the results of the Package Sanity Test.

### Screenshots

Successful WACK test with WindowsAppSDK v1.4

![image](https://github.com/microsoft/WindowsAppSDK/assets/78753788/e53cc15f-f3b3-4b0f-aa24-7f5f1629bef5)

Failed WACK test with WindowsAppSDK v1.5

![image](https://github.com/microsoft/WindowsAppSDK/assets/78753788/24df8fe3-1450-48eb-9ae9-cd959fc53af2)

### NuGet package version

Windows App SDK 1.5.1: 1.5.240311000

### Packaging type

Packaged (MSIX)

### Windows version

Windows 10 version 22H2 (19045, 2022 Update)

### IDE

Visual Studio 2022

**Comment by greatoceansoftware** [comment]
Any update on this? I have tried a few ideas from similar threads on this topic, but still unable to pass the test.

**Comment by greatoceansoftware** [comment]
I was able to work around this. I basically had to recreate the project from scratch, which produced a MUCH different csproj file. Here are the steps taken. I won't close this until my next successful MS Store publication, which could be a few weeks.

- Create new project.
- Manually edit csproj file (for .NET 8 compatibility)

```
    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
    <PublishProfile>win-$(Platform).pubxml</PublishProfile>
```

- Set other project properties.
- Import source code.
- Build and test.
- Publish to sideloading MSIX and WACK test.
- Reassociate with MS Store.
- Publish to MS Store MSIX, recreating assets.
- WACK test.

**Comment by greatoceansoftware** [comment]
Closing this issue since another issue with v1.5 is causing me to wait on 1.6.

https://github.com/microsoft/microsoft-ui-xaml/issues/9553

I'll continue with v1.4 until then.
